docs/DOCKER.md

```
@@ -261,79 +281,60 @@ docker exec gems-postgres-prod pg_dump -U gems gems_of_india > backup.sql
-docker volume ls
-```
-
-### Backup Volumes
+After seeding, you can use these test accounts (passwords would need to be set via your auth system):
 
-```bash
-# Backup PostgreSQL data
-docker run --rm -v gems-of-india_postgres_data:/data -v $(pwd):/backup alpine tar czf /backup/postgres-backup.tar.gz -C /data .
+| Name         | Email                    | Role      |
+| ------------ | ------------------------ | --------- |
+| Rajesh Kumar | rajesh.kumar@example.com | Admin     |
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)